### PR TITLE
Add missing @PathVariable declarations in examples in webmvc.adoc

### PR DESCRIPTION
**Pull request by swapy**
We say
 For example, given URL "/spring-web-3.0.5 .jar", the following method extracts the name, version, and file extension:

So added PathVariable name to complete the code.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@swapy Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-framework&pullRequestId=25006)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-framework&pullRequestId=25006) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@swapy Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-framework&pullRequestId=25006)!

**Comment by sbrannen** [comment]
Thanks for the PR. We'll get this merged soon.

File: src/docs/asciidoc/web/webmvc.adoc [hunk 1]
```diff
@@ -1634,15 +1634,15 @@ extracts the name, version, and file extension:
 .Java
 ----
 	@GetMapping("/{name:[a-z-]+}-{version:\\d\\.\\d\\.\\d}{ext:\\.[a-z]+}")
-	public void handle(@PathVariable String version, @PathVariable String ext) {
+	public void handle(@PathVariable name, @PathVariable String version, @PathVariable String ext) {
```
Comment (sbrannen) [inline src/docs/asciidoc/web/webmvc.adoc:1637]: FYI: you accidentally deleted the `String` type declaration for the `name` parameter.

But don't worry about it: I've fixed it locally prior to merging.

```suggestion
	public void handle(@PathVariable String name, @PathVariable String version, @PathVariable String ext) {
```

Comment (swapy) [reply src/docs/asciidoc/web/webmvc.adoc:1637]: Sorry for that and thanks a lot. I will be careful next time.

**Pull request closed without merging**